### PR TITLE
fix: path to head assembly 3d parts

### PR DESCRIPTION
**Pull request by sethfischer**
Fix path to head Head Assembly 3D parts and make the path a link.

### CODE REVIEW

File: mechanical/head_assembly/README.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ The head assembly serves as the head and face of our rover. It houses a 16x32 fu
 
 
 ## 3. 3D Printing
-There are a few components that need to be 3D printed to make the head assembly. You can find the STL files necessary for these prints in the ”Mechanical/Head Assembly/3D Printed Parts” folder of the repository.
+There are a few components that need to be 3D printed to make the head assembly. You can find the STL files necessary for these prints in the [mechanical/head_assembly/3d_printed_parts](https://github.com/nasa-jpl/open-source-rover/tree/master/mechanical/head_assembly/3d_printed_parts) folder of the repository.
```
Comment (Achllle) [inline mechanical/head_assembly/README.md:45]:
```suggestion
There are a few components that need to be 3D printed to make the head assembly. You can find the STL files necessary for these prints in the [mechanical/head_assembly/3d_printed_parts](3d_printed_parts) folder of the repository.
```
Relative link should work as well

**Review by Achllle — COMMENTED** [review]
Suggest using a relative link but looks good otherwise

File: mechanical/head_assembly/README.md (hunk 1 above)
Comment (sethfischer) [reply mechanical/head_assembly/README.md]: @Achllle Link updated

**Comment by ericjunkins** [comment]
LGTM

**Pull request merged**